homework/style.css

```diff
@@ -1,3 +1,78 @@
+header {
+    background-color: rgb(116, 56, 13); 
+    color: white; 
+    padding: 1rem; 
+}
+
+body {
+    background-color:rgb(197, 197, 197);
+    color: black;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size: 1rem;
+    padding-right: 1px;
+}
+
 .alert-error {
     color: red;
+}
+
+
+select {
+    font-size: .9rem;
+    padding: 4px 25px;
+    margin-left: 10px;
+    border-radius: 20px;
+}
+
+#container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+
+    margin: 1 rem;
+}
+
+#forRepoBlock {
+    color: rgb(43, 0, 43);
+    font-size: .9rem;
+    padding: 4px 4px;
+    margin-left: 2px;
+    margin-top: 1rem;
+    width: 60%;
+
+    border-style: bold;
+    border-radius: 0%;
+    padding: 10px;
+    background:rgba(230, 178, 210, 0.5);
+    opacity: 1;
+
+    box-shadow: 10px 5px 5px grey;
+}
+#forContributorsBlock {
```

Comment:
I'm kind of following what you want to do here, but would prefer another solution (this one doesn't use semantic html and isn't easy to understand):

I'm assuming a html-strucuture like this:
```html
<div class="container contributors">
  <h2>Contributors</h2>
  <ul class="forContributorsBlock">
    <li class="contributor">
      <a href="...">contributor name</a>
      <img src="..." />
      <span>5</span>
    </li>
  </ul>
</div>
```

```css
.contributors {
 color: rgb(48, 0, 20);
 margin: ...;
 padding: ...;
 border: ...;
}

.forContributorsBlock {
 list-style: none;
}
```

Reply:
Oh, yes that would be awesome to achieve. I think I still need to practice more DOM manipulations to be able to complete this one :(